### DISCUSSION TITLE
@ bluebill1049, How many hours do you sleep a day 😂

### DISCUSSION THREAD

**Discussion by chshouyu** [react-hook-form/react-hook-form#2626] [General] [answered]
How do you keep so little open issues and answer in Discussions so timely?
It's awesome!

**Comment by bluebill1049** [comment, marked as answer]
lol I am actually quite sleepy right now 😭

**Reply by bluebill1049** [reply]
getting a ☕ now.

**Reply by chshouyu** [reply]
Does RHF is a Full-time development project?

**Reply by bluebill1049** [reply]
no :( I work at nights and weekends 👍
